core/src/test/java/org/infinispan/conflict/impl/DenyReadWriteRemoveAllTest.java

```diff
@@ -27,10 +30,17 @@
        * N => total number of nodes
        */
       return new Object[] {
-            new DenyReadWriteRemoveAllTest().setPartitions("AD-5N", AvailabilityMode.DEGRADED_MODE, new int[]{0,1,2}, new int[]{3,4}),
-            new DenyReadWriteRemoveAllTest().setPartitions("AD-4N", AvailabilityMode.DEGRADED_MODE, new int[]{0,1}, new int[]{2,3}),
-            new DenyReadWriteRemoveAllTest().setPartitions("PD-4N", new int[]{0,1,2}, new int[]{3}),
-            new DenyReadWriteRemoveAllTest().setPartitions("PD-3N", new int[]{0,1}, new int[]{2}),
+            new DenyReadWriteRemoveAllTest().init(REPL_SYNC, "PD-5N", new int[]{0,1,2}, new int[]{3,4}),
+            new DenyReadWriteRemoveAllTest().init(REPL_SYNC, "AD-4N", DEGRADED_MODE, new int[]{0,1}, new int[]{2,3}),
+            new DenyReadWriteRemoveAllTest().init(REPL_SYNC, "PD-4N", new int[]{0,1,2}, new int[]{3}),
+            new DenyReadWriteRemoveAllTest().init(REPL_SYNC, "PD-3N", new int[]{0,1}, new int[]{2}),
+            new DenyReadWriteRemoveAllTest().init(REPL_SYNC, "AD-2N", DEGRADED_MODE, new int[]{0}, new int[]{1}),
+
+            new DenyReadWriteRemoveAllTest().init(DIST_SYNC, "AD-5N", DEGRADED_MODE, new int[]{0,1,2}, new int[]{3,4}),
+            new DenyReadWriteRemoveAllTest().init(DIST_SYNC, "AD-4N", DEGRADED_MODE, new int[]{0,1}, new int[]{2,3}),
+            new DenyReadWriteRemoveAllTest().init(DIST_SYNC, "PD-4N", new int[]{0,1,2}, new int[]{3}),
+            new DenyReadWriteRemoveAllTest().init(DIST_SYNC, "PD-3N", new int[]{0,1}, new int[]{2}),
+            new DenyReadWriteRemoveAllTest().init(DIST_SYNC, "AD-2N", DEGRADED_MODE, new int[]{0}, new int[]{1}),
```

Comment:
Couldn't the `init` method be replaced with a constructor?

Reply:
Yes, I'll do that. Previously there were several overloaded variations of `init` that I did not want to have to create multiple constructors on the sublcasses for, but that's no longer the case.